.github/workflows/dockerhub-readme.yml

```diff
@@ -0,0 +1,64 @@
+name: Generate DockerHub READMEs
+
+'on':
+  push:
+    branches:
+      - main
```

Comment:
This is tricky. `main` is used for all of our stable releases. However, the `README.md`s can change in between stable releases (i.e. for the nightlies) and we'd still want this GitHub Action to trigger. Ideally, we'd set this to be our default branch (i.e. `branch-0.17`), but this is problematic since our default branch changes regularly. We should see if there's some way for us to fix this.

Reply:
I think we could get away with using `branches: branch-*` for the branch here. The `rapidsai/docker` repo doesn’t follow the same burndown process as the library repos (where `branch-0.17` and `branch-0.18` can be active at the same time). So since this repo only ever has a single active `branch-0.xx` branch, this should probably suffice.

Reply:
we could listen to every push, check default branch with git and abort on that. it will be spammy though.

Reply:
updated with `branches-*`